[WIP] Allow admin to show/generate ticket

siddhantbajaj opened the pull request:
![screen shot 2017-06-26 at 4 08 08 am](https://user-images.githubusercontent.com/17237680/27520286-20d7a180-5a25-11e7-9736-f007922f4efc.png)

Ana06 commented on app/views/layouts/_admin_sidebar.html.haml:
```diff
@@ -139,6 +139,11 @@
       = link_to(admin_conference_roles_path(@conference.short_title)) do
         %span.fa.fa-group
         Roles
+  - if can? :index, PhysicalTicket.new(conference: @conference)
```
I would say this is not the best place for that. We have a donation section, which has tickets on it. It could be a button inside the ticket (which is inside donations) section. I think we don't need to add this to the sidebar, which is getting quite long. :thinking:

siddhantbajaj replied:
👍

Ana06 commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Export ticket to pdf](https://trello.com/c/cwXizd5S/6-export-ticket-to-pdf)

siddhantbajaj commented:
Ticket Index page

1. Moved Ticket sold distribution and ticket turnover distribution to physical ticket index page.
2. Added link for physical ticket index page.

Ana06 commented on app/views/admin/physical_ticket/index.html.haml:
```diff
@@ -0,0 +1,41 @@
+              %tr
+                %td= physical_ticket.id
+                %td= physical_ticket.ticket.title
+                %td= physical_ticket.user.name
```
I wonder what would be more useful, I would say the email would be better. :thinking:

siddhantbajaj commented:
@Ana06 
> why did you need to regenerated haml-lint_todo.yml?

Line 10,12 were long, but found a better fix so no need to regenerate haml-lint_todo now.

Ana06 commented:
LGTM :tada: 

@mdeniz please take a look! :wink:

mdeniz commented:
Nice work!!! 👍
